app/controllers/shopify_app/callback_controller.rb

```diff
@@ -82,7 +86,7 @@ def install_scripttags
 
       ScripttagsManager.queue(
         shop_name,
-        token,
+        shop_session&.token || user_session.token,
```

Comment:
Should we use the safe navigator for the user token as well? I believe the session can be `nil` too.